docs/internal/dev/system-architecture.md

```
@@ -575,59 +468,51 @@ Non-cron periodic tasks running in the server process.
 | WebSocket & Real-Time     | 4     | DevServerService, NotificationService, ActionableItemService, ActionableItemBridge                                                    |
 | Settings & Context        | 4     | SettingsService, UserIdentityService, ContextAggregator, SensorRegistryService                                                        |
 | Communication & Routing   | 4     | ChannelRouter, EscalationRouter, SignalIntakeService, HITLFormService                                                                 |
-| Review & Feedback         | 4     | PRFeedbackService, AntagonisticReviewService, WorktreeLifecycleService, ReconciliationService                                         |
+| Review                    | 3     | AntagonisticReviewService, WorktreeLifecycleService, ReconciliationService                                                            |
 
 ### Wiring Order
 
-Services are wired in strict order via 15 register modules in `server/wiring.ts`:
+Services are wired in strict order via register modules in `server/wiring.ts`:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)